README.md

```diff
@@ -51,6 +51,7 @@ The response will have some additional headers:
 |`x-ratelimit-limit`     | how many request the client can do
 |`x-ratelimit-remaining` | how many request remain to the client in the timewindow
 |`retry-after`           | if the max has been reached, the millisecond the client must wait before perform new requests
+|`x-ratelimit-reset`     | how many seconds must pass before the rate limit resets (**only** with the redis setup)
```

Comment:
Is there some limitations on why we can't offer this without Redis?

Reply:
In the local storage, we are setting up an interval that clear the cache every `timeWindow` sec

So, for what I know, there is not the possibility to get how many ms are lacking to the next interval trigger. 
The solution could be to have a sibling interval that counts the seconds and will be reset together with the cache: could fit this case?

Reply:
You'd need to record the time the last interval has fired, and then count how much time is left.

Reply:
It was obvious 🤯

I will add it 👍

Reply:
Done!